scripts/app.js

```diff
@@ -327,7 +327,9 @@ const activateState = (model, id) =>
 const startRecipe = (model, id, name) =>
   batch(update, model, [
     SetRecipeForEnvironment(id, name),
-    PostStopStartRecipe(model.environment.id, id),
+    model.recipes.active === null ?
```

Comment:
It should compile now. however I think the state passed to `const startRecipe` is already updated with the active recipe that the user just selected, so there is no way to test there if there was a running recipe _previously_. I guess this branching (start vs stop-start) should be handled earlier in execution. 
Any ideas?